### DISCUSSION TITLE
安卓RustDesk无法远程桌面pc端

### DISCUSSION THREAD

**Discussion by heiyejiang** [rustdesk/rustdesk#846] [Q&A] [answered]
安卓版的RustDesk如果和pc端在同一局域网中能够正常远程桌面，但是如果安卓手机和pc端不在同一局域网就无法远程桌面pc端；会报错“连接错误 连接被对方关闭”

**Comment by rustdesk** [comment, marked as answer]
https://github.com/rustdesk/rustdesk/issues/499

**Reply by heiyejiang** [reply]
感谢大佬的恢复；问题已经解决了，我把中继服务器的端口填写成21116了；因为之前看您的文档上https://rustdesk.com/docs/zh-cn/self-host/install/#%E6%AD%A5%E9%AA%A43-%E5%9C%A8%E5%AE%A2%E6%88%B7%E7%AB%AF%E8%AE%BE%E7%BD%AE-hbbshbbr-%E5%9C%B0%E5%9D%80填写的端口就是21116所以我也跟到填21116其实在安装hbbr 时监听的是21117(tcp)
